tcpdf.php

```diff
@@ -2935,6 +2943,18 @@ public function SetCreator($creator) {
 		$this->creator = $creator;
 	}
 
+	/**
+	 * Whether to allow local file path in image html tags, when prefixed with file://
+	 * 
+	 * @param $allowLocalFiles (bool) True, when local files should be allowed. Otherwise false.
+	 * @public
+	 * @since 6.4
+	 */
+	public function SetAllowLocalFiles($allowLocalFiles) {
```

Comment:
```suggestion
	public function setAllowLocalFiles($allowLocalFiles) {
```

Reply:
all setters within this file begin with a uppercase "S". I tried to be consistent.

Reply:
Oh okay, I did not know that.